wellcomecollection/wellcomecollection.org
# Track with Hotjar on Jason only, on stopId pages and DG landing

## What does this change?

Relates to #11131 kind of, in which we add tracking to the new [stopId] pages as well as  the `/guides/exhibitions/[exhibitionId]` page, but only if it's Jason's 👀 

> [!NOTE]
> Sorry, some linting import changes have made their way here as well, so some import changes added.

## How to test

Confirm I've used the correct ID on these and run locally to check that it works.

## How can we measure success?

More tracking info = better future, I guess? 😄 

## Have we considered potential risks?
None really.
